apps/application/chat_pipeline/step/chat_step/impl/base_chat_step.py

```diff
@@ -214,7 +214,7 @@ def get_block_result(message_list: List[BaseMessage],
                 'status') == 'designated_answer':
             return AIMessage(no_references_setting.get('value').replace('{question}', problem_text)), False
         if chat_model is None:
-            return AIMessage('抱歉，没有配置 AI 模型，无法优化引用分段，请先去应用中设置 AI 模型。'), False
+            return AIMessage('抱歉，没有配置 AI 模型，请先去应用中设置 AI 模型。'), False
         else:
             return chat_model.invoke(message_list), True
 
```

Comment:
The provided code looks generally clean and follows proper structure. However, there is one potential issue with variable names:

```python
chat_model is None
```

In Python, `None` cannot be used on the left side of an assignment because it's not a valid identifier. Typically, you would use `not ...` to check if something is `None`. Here’s how you can fix it:

**Change from:**
```python
if chat_model is None:
```
**To:**  
```python
if not chat_model:
```

This change makes the code more readable and maintains idiomatic usage in Python.

Here is the adjusted code snippet:

```python
@@ -170,8 +170,8 @@
         return iter(
             [AIMessageChunk(content=no_references_setting.get('value').replace('{question}', problem_text))]), False
     if not chat_model:
-        return iter([AIMessageChunk('抱歉，没有配置 AI 模型，无法优化引用分段，请先去应用中设置 AI 模型。')]), False
+        return iter([AIMessageChunk('抱歉，没有配置 AI 模型，请先到应用中设置 AI 模型。')]), False
     else:
         return chat_model.stream(message_list), True

@@ -214,8 +214,8 @@
         return AIMessage(no_references_setting.get('value').replace('{question}', problem_text)), False
     if not chat_model:
-        return AIMessage('抱歉，没有配置 AI 模型，无法优化引用分段，请先去应用中设置 AI 模型。'), False
+        return AIMessage('抱歉，没有配置 AI 模型，请先到应用中设置 AI 模型。'), False
     else:
         return chat_model.invoke(message_list), True
``` 

So that your condition now reads correctly and is both idiomatically correct and free from syntax errors in Python 3.x.